### DISCUSSION TITLE
Caching breaks dependency symbols

### DISCUSSION THREAD

**Discussion by shyim** [JetBrains/Qodana#139] [Q&A]
Without cache anything works as expected with Qodana I have 0 issues:
https://github.com/shyim/danger-php/actions/runs/3650042109/jobs/6165579204

But on the next run any dependency cannot be found anymore.

https://github.com/shyim/danger-php/actions/runs/3650042109

So I disabled caching in the Action, and it works now flawlessly. https://github.com/shyim/danger-php/commit/6e86f38acaf687c7615d5dc63845753437e1987c

Is maybe something not cached correct with the Action?

**Comment by skasi-surya** [comment]
You need to add Github's [Cache](https://github.com/actions/cache) action before the Qodana action in your workflow. I don't use PHP but an example can be found [here](https://github.com/actions/cache/blob/main/examples.md#php---composer).

**Reply by shyim** [reply]
There is nothing in my qodana yaml what caches? https://github.com/shyim/danger-php/blob/6e86f38acaf687c7615d5dc63845753437e1987c/.github/workflows/qodana.yml
